tests/ludwig/encoders/test_sequence_encoders.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
```

Comment:
Could you also include changes in `tests/integration_tests/test_sequence_encoders.py` into this file, followed by removing `tests/integration_tests/test_sequence_encoders.py`?

Reply:
Lets make that a separate PR - though these tests are overlapping a lot they are testing different interfaces so I'd like to make that a separate change.

Reply:
Sounds good!